feat(curriculum): add block 2 dialogs and tasks of the B1 course of the English Curriculum (#57471)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f76b915c1cf062570fdfc.md

---
id: 671f76b915c1cf062570fdfc
title: Task 95
challengeType: 19
dashedName: task-95
---

<!-- (Audio) Brian: Did you go to the AI workshop yesterday? Maria: Yes, I did. -->

# --instructions--

Listen to the audio and answer the question below.

# --questions--

## --text--

Which workshop can we be sure Maria attended yesterday?

## --answers--

The Python workshop.

### --feedback--

Maria specifically confirms attending the AI workshop, not the Python workshop.

---

The AI workshop.

---

The cybersecurity workshop.

### --feedback--

Maria attended the AI workshop, as confirmed in her response.

---

The design workshop.

### --feedback--

Maria explicitly says she attended the AI workshop, not the design workshop.

## --video-solution--

2

# --explanation--

> AI stands for `Artificial Intelligence`, a field of computer science that focuses on creating systems capable of performing tasks that typically require human intelligence, such as learning, reasoning, and decision-making. Example: `AI is being used to improve natural language processing in chatbots.` (this means that Artificial Intelligence is used to help chatbots communicate in a more efficient way with humans) 
> 
> An AI workshop is a session or event where participants learn about AI concepts, tools, or applications. These workshops often include hands-on activities to help attendees understand and implement AI technologies. Example: `The AI workshop covered topics like machine learning and neural networks.` (this means that, if you attend an AI workshop, you may learn by practicing with things like machine learning and neural networks) 
> 
> In this dialog, Maria confirms that she attended an AI workshop yesterday.

Reviewer: ```suggestion
AI stands for `Artificial Intelligence`, a field of computer science that focuses on creating systems capable of performing tasks that typically require human intelligence, such as learning, reasoning, and decision-making. For example: 

`AI is being used to improve natural language processing in chatbots.` - This means that Artificial Intelligence is used to help chatbots communicate in a more efficient way with humans.

An `AI workshop` is a session or event where participants learn about AI concepts, tools, or applications. These workshops often include hands-on activities to help attendees understand and implement AI technologies. For example: 

`The AI workshop covered topics like machine learning and neural networks.` - This means the AI workshop taught machine learning and neural networks.

In this dialog, Maria confirms that she attended an AI workshop yesterday.
```
